app/classes/generate_new_week_assignments.rb

```diff
@@ -1,5 +1,5 @@
-module AssignmentGenerationHelper
-  def generate_new_week_assignments(week)
+class GenerateNewWeekAssignments
```

Comment:
Excellent!

Let's move this from `app/classes/` which is too vague to be a helpful namespace to `app/services/` (some devs might use `app/interactors/`).